templates/docs/examples/patterns/tables/table-mobile-card.html

```diff
@@ -49,16 +49,16 @@
       </td>
     </tr>
     <tr>
-      <td aria-label="FQDN">upward-muskox</td>
-      <td aria-label="Power">Off</td>
-      <td aria-label="Status">Allocated</td>
-      <td aria-label="Owner">sparkiegeek</td>
-      <td aria-label="Zone">london</td>
-      <td class="u-align--right"  aria-label="Cores">6</td>
-      <td class="u-align--right"  aria-label="RAM">4 GiB</td>
-      <td class="u-align--right"  aria-label="Disks">1</td>
-      <td class="u-align--right"  aria-label="Storage this is long enough to cause wrapping">500GB</td>
-      <td class="has-overflow" aria-label="Actions">
+      <td data-heading="FQDN">upward-muskox</td>
+      <td data-heading="Power">Off</td>
+      <td data-heading="Status">Allocated</td>
+      <td data-heading="Owner">sparkiegeek</td>
+      <td data-heading="Zone">london</td>
+      <td class="u-align--right"  data-heading="Cores">6</td>
+      <td class="u-align--right"  data-heading="RAM">4 GiB</td>
+      <td class="u-align--right"  data-heading="Disks">1</td>
+      <td class="u-align--right"  data-heading="Storage this is long enough to cause wrapping">500GB</td>
```

Comment:
It seems we no longer wrap these headings, so this label doesn't make sense anymore

```suggestion
      <td class="u-align--right"  data-heading="Storage">500GB</td>
```